tiledb/tests/test_pandas_dataframe.py

```diff
@@ -469,4 +469,27 @@ def test_csv_chunked(self):
             res = A.multi_index[int(ned[0]):int(ned[1])]
             df_bk = pd.DataFrame(res)
 
+            tm.assert_frame_equal(df_bk, df)
+
+    def test_csv_fillna(self):
+        col_size = 10
+        data = np.random.rand(10)
+        data[4] = np.nan
+        df = pd.DataFrame({'v': data})
+
+        tmp_dir = self.path("csv_fillna")
+        os.mkdir(tmp_dir)
+        tmp_csv = os.path.join(tmp_dir, "generated.csv")
+
+        df.to_csv(tmp_csv, index=False)
+
+        tmp_array = os.path.join(tmp_dir, "array")
+        tiledb.from_csv(tmp_array, tmp_csv, fillna={'v': 0})
+
+        df['v'][4] = 0
```

Comment:
Should this be an assert?

Reply:
No, this updates the value to what is expected on read-back because of the fill. Otherwise the `assert_frame_equal` will fail.